travis-dependencies.sh

```diff
@@ -1,6 +1,8 @@
 #!/bin/bash
 
 if [[ $TRAVIS_OS_NAME == "osx" ]] ; then
+  brew install ccache                       # enable caching on mac
+  PATH=/usr/local/opt/ccache/libexec:$PATH  # https://docs.travis-ci.com/user/caching/#ccache-on-OS-X
```

Comment:
Do we need brew update for sure? It has a *huge* impact on build time (2min!).
https://docs.travis-ci.com/user/installing-dependencies/#Installing-Packages-on-OS-X